frontend/src/bundles/video_editor/components/timeline/subcomponents/item.tsx

```diff
@@ -0,0 +1,41 @@
+import { type Span } from 'dnd-timeline';
+import { useItem } from 'dnd-timeline';
```

Comment:
It's better to not duplicate imports and have them in one line

Reply:
Solved, thank you.